src/main/java/org/opensearch/performanceanalyzer/collectors/NodeStatsFixedShardsMetricsCollector.java

```diff
@@ -106,42 +106,6 @@ private void populateCurrentShards() {
                     put(
                             ShardStatsValue.SEGMENTS_TOTAL.toString(),
                             (shardStats) -> shardStats.getStats().getSegments().getCount());
-                    put(
-                            ShardStatsValue.SEGMENTS_MEMORY.toString(),
```

Comment:
Do we want to remove the related Metrics as well?

Reply:
Like from RCA package or something else? I will get out a separate PR for RCA related changes.